test/unit/lodash.spec.js

```diff
@@ -0,0 +1,800 @@
+        //         var param1 = params[0];
+        //         for(var i=0; i < param1.length; i++){
+        //             expect(param1[i].length).to.eql(2);
+        //         }
+        //     });
+        // });
+    });
+
+    // ----------------------- STRING ----------------------- //
+
+    describe('template', function() {
+        beforeEach(function() {
+            params = datasets.template();
+        });
+        it('should match types of passing elements', function() {
+            var param1 = params[0];
+            var param2 = params[1];
+            expect(param1).to.be.a('string');
+            expect(params).to.have.length.within(1,2);
+            if(param2 !== undefined && params.length == 2){
+                expect(param2).to.be.an('object');
+                expect(Object.keys(param2)).to.have.length.within(1,6);
+            }
+        });
+        it('should check options', function() {
+            var options = params[1];
+            expect(options).to.have.any.keys('escape', 'evaluate', 'imports' ,'interpolate', 'sourceURL', 'variable');
+            for(var key in options){
+                // expect(key).to.be.oneOf(['escape', 'evaluate', 'imports' ,'interpolate', 'sourceURL', 'variable']);
+                if(key == 'escape'){
+                    expect(key).to.be.a('RegExp');
+                }
+                if(key == 'evaluate'){
+                    expect(key).to.be.a('RegExp');
+                }
+                if(key == 'imports'){
+                    expect(key).to.be.an('Object');
+                }
+                if(key == 'interpolate'){
+                    expect(key).to.be.a('RegExp');
+                }
+                if(key == 'variable'){
+                    expect(key).to.be.a('string');
+                }
+                if(key == 'variable'){
+                    expect(key).to.be.a('string');
+                    expect(options[key]).to.not.be.empty;
+                }
+            }
+        });
+        it('should return array property names of functions', function() {
+            // expect(_.template.apply(_, params)).to.eql(true);
+        });
+    });
+
+    // ----------------------- OBJECTS ----------------------- //
+
+    describe('functions', function() {
+        beforeEach(function() {
+            params = datasets.functions();
+        });
+        it('should match types of passing elements', function() {
+            var obj = params[0];
+            expect(obj).to.be.an('object');
+            expect(params).to.have.length(1);
+        });
+        it('should contain only one property square', function() {
+            var obj = params[0];
+            var i = 0;
+            for (var key in obj) {
+                if (obj.hasOwnProperty(key) && key === 'square') {
+                    if (obj[key] instanceof Function) {
+                        i++;
+                    }
+                }
+            }
+            expect(i).to.eql(1);
```

Comment:
Check chai documentation. For that case you should've just provided a correct expectation(.ownProperty(name)).